test/integration/integration_test_fixtures.py

```diff
@@ -21,6 +19,15 @@
 
 
 def determineIpVersionsFromEnvironment():
+  """Determine the IP version(s) for test execution from the environment.
+
+  Raises:
```

Comment:
We should only add the Raises section for exceptions raised by the function directly. Otherwise this is a futile effort since every line and character of Python code can raise at least one exception (SyntaxError, MemoryError, RuntimeError, ...).

This functions doesn't have the raise statement, so it shouldn't need the Raises section.

Reply:
`determineIpVersionsFromEnvironment()` has:

```
  else:
    raise NighthawkException("Unknown ip version: '%s'" % versions)

```

So I think in this instance it's OK?